front/components/Dashboard/DashboardHelptext.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import '../../testSetup';
+
+import { shallow } from 'enzyme';
+import * as React from 'react';
+
+import { videoState } from '../../types/Video';
+import { DashboardHelptext } from './DashboardHelptext';
+
+describe('<DashboardHelptext />', () => {
+  it('displays the relevant helptext for each state', () => {
+    expect(
+      shallow(<DashboardHelptext state={videoState.ERROR} />).html(),
+    ).toContain('There was an error with the video');
+    expect(
+      shallow(<DashboardHelptext state={videoState.PENDING} />).html(),
+    ).toContain('There is currently no video for this location');
```

Comment:
What do you mean by "location" in this context?

Reply:
"Instance of the xblox" as in "location in the page you're building"

If you have a better way to express this I'll gladly change it

Reply:
Oh I see. What about:There is currently no video for this location

Reply:
Perfect! Thanks